docs/send-data/opentelemetry-collector/troubleshooting.md

```diff
@@ -152,17 +152,16 @@ After running the collector for the first time, changes to collector properties
 
 To work around this, you need to delete the existing collector registration and register the collector again. To do this, you need to do two things:
 
+1. Delete the local collector registration file in `~/.sumologic-otel-collector/` and wait for 10 minutes for the collector to get offline.
 1. Remove the collector in Sumo Logic UI.
-   1. Log in to Sumo Logic.
-   1. Go to `Manage Data` - `Collection`.
+   1. Sign in to Sumo Logic platform.
+   1. Navigate to **Manage Data** > **Collection** > **OpenTelemetry Collection**.
```

Comment:
This should 2.

Reply:
@Gourav2906, in our markdown, although steps are labelled as "1." they render as numbered lists. See "[Numbered Lists](https://help.sumologic.com/docs/contributing/style-guide/#numbered-lists)" in our style guide.